learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,85 @@
-#  [Learning Bitmark / Quick start] - Simple solution for node setup
+#  [ Learning Bitmark / Quick start ] - Simple solution for node setup
+
+## Start your bitmark node
+
+A user can run a bitmark node in several ways. This section will guide you to start a bitmark node using docker container. Inside a bitmark node docker container, there are bitmarkd, recorderd, payment system and a user interface to control and monitor services.
+
```

Comment:
And I see I missed your question here, about which is best: "option 1 A user can run a bitmark node in several ways.
option 2 You can run a bitmark node in several ways.
@nodestory: I personally would like to avoid both options. My option is
There are several ways to run a bitmark node."

The last one is passive voice, where you don't have an active subject. I would usually avoid it unless there are no options (and sometimes there aren't).

The other two are both fine. Saying "A user" is more formal and "you" is more casual. I personally like "you" because it's very direct. The biggest thing is just to be consistent. If you regularly address the readers as "a user" (or other noun), don't use "you", and if you regularly use "you", use it whenever you're directly addressing the readers. So, make a style decision, and we can just ensure that it's used.